app/bundles/LeadBundle/Form/Type/DashboardSegmentContactsWidgetType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Mautic\LeadBundle\Form\Type;
+
+use Mautic\LeadBundle\Model\ListModel;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @extends AbstractType<mixed>
+ */
+class DashboardSegmentContactsWidgetType extends AbstractType
```

Comment:
All new classes should be final by default
```suggestion
final class DashboardSegmentContactsWidgetType extends AbstractType
```